config/locales/en.yml

```diff
@@ -450,11 +450,30 @@ en:
         signed_on: Signed on %{date}
   home:
     show:
+      about_card_1_content: RailsDevs strives to give power back to the independent developer. Instead of companies posting their jobs, developers post their profiles.
+      about_card_1_title: Empowering the developer
+      about_card_2_content: RailsDevs is an Open Startup and open source. It operates fully transparent and shares its metrics, like revenue and traffic.
+      about_card_2_title: Doing everything in public
+      about_card_3_content: The platform is open to all developers, regardless of race, color, national origin, religion, gender, age, disability, or skill level that follow the code of conduct.
+      about_card_3_title: Creating a safe, inclusive environment
+      about_subtitle_1: RailsDevs is making it easier for Rails developers to find their next gig.
+      about_subtitle_2: It's built around three core values.
+      about_title: About RailsDevs
       available_developers: Developers available now
+      bottom_of_page_cta_1: Find
+      bottom_of_page_cta_2: your next
+      bottom_of_page_cta_3: Rails gig.
       description_og: RailsDevs empowers independent developers available for their next gig. Stop scouring job boards and sit back as companies reach out to you first.
+      explain_title: How it Works
       get_started: Get started
+      hero_title_1: The
+      hero_title_2: job board for Rails developers
+      hero_title_accent: reverse
       learn_more: Learn more
       more: See more developers
+      reviews_header: What Others are Saying
```

Comment:
```suggestion
      reviews_header: What others are saying
```

Reply:
@joemasilotti As a general rule, do you prefer sentence case?

Reply:
Yep!